src/test/java/be/seeseemelk/mockbukkit/entity/TNTPrimedMockTest.java

```diff
@@ -0,0 +1,91 @@
+package be.seeseemelk.mockbukkit.entity;
+
+import be.seeseemelk.mockbukkit.MockBukkit;
+import be.seeseemelk.mockbukkit.ServerMock;
+import org.bukkit.entity.EntityType;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class TNTPrimedMockTest
+{
+	private TNTPrimedMock tntPrimed;
+	private ServerMock server;
+
+	@BeforeEach
+	void setUp()
+	{
+		server = MockBukkit.mock();
```

Comment:
Please look at the other Test files. This can be places by using our `MockbukkitExtension` class and the `@MockbukkitInject` Annotation

Reply:
I looked other tests and other entity tests don't use MockbukkitExtension